governance/executive-council.md

```diff
@@ -0,0 +1,262 @@
+# pyOpenSci Executive Council
+
+The pyOpenSci Executive Council (EC) has four primary functions:
+
+1. **Guides the Organization**: The Executive Council defines and directs pyOpenSci’s high-level [mission, vision, and values](mission-values) and sets its strategic direction.
+
+2. **Advocates for pyOpenSci**: The Executive Council serves as an advocate and ambassador for the organization, leveraging members’ networks to enhance its reputation and support fundraising efforts.
+
+3. **Provides Strategic Insight**: The Executive Council offers perspective and guidance on major strategic decisions to ensure the organization’s long-term success.
+
+4. **Holds the [Executive Director](executive-director) accountable**: The Executive Council holds the Executive Director accountable while also providing high-level strategic and financial guidance and oversight.
+
+The Executive Director is currently a voting member of the Executive Council for items unrelated to their salary and accountability.
+
+The Executive Council holds broad expertise in building and running a scientific / data science nonprofit organization. It is complemented by the [Advisory Council](advisory-council), which offers domain-specific expertise, guidance, and input.
+
+## How the Executive Council works
+
+The pyOpenSci Executive Council operates through open discussions, collaborative
+decision-making, and a commitment to transparency. By leveraging diverse
+perspectives, the Council ensures that its guidance aligns with the organization’s
+[mission, vision, and values](mission-values) while meeting the needs of the
+growing pyOpenSci community.
+
+The Council employs flexible and adaptive practices, such as the **lazy consensus**
+approach, to ensure decisions are made efficiently while fostering thoughtful
+dialogue and collective alignment.
+
+## Executive Council Roles & Responsibilities
+
+Each Executive Council (EC) member dedicates no more than **four to six hours per month** to pyOpenSci activities, including meeting attendance. Executive Council members are expected to fulfill the following responsibilities:
+
+- Attend at least 80% of monthly Executive Council meetings.
+- Participate in asynchronous commitments, which include:
+  - Suggesting agenda items for meetings.
+  - Communicating with the Executive Director (ED) via Slack.
+  - Providing timely reviews of materials and documents.
+  - Connecting the Executive Director with potential funding opportunities.
+- Evaluate the performance of the Executive Director on a yearly basis.
+- Approve expense reports.
+- Take meeting minutes and ensure they are [published in this Handbook](ec-minutes).
```

Comment:
change to published in the [Executive Council Minutes section of the Handbook](link)

Reply:
More specific. I made this change locally.